app/views/catalog/_show_actions_box_default.html.erb

```diff
@@ -21,7 +21,11 @@
 
   <% if requestable %>
     <div class='mr-auto al-action-box-request'>
+      <% if ead_files(document).present? %>
```

Comment:
Should we extract lines 22-30(?) to a separate partial? This might make it easier for downstream implementors to customize their request behavior.